templates/includes/sidebar.html

```diff
@@ -0,0 +1,15 @@
+<ul>
+  <li>
+    <a href="#">Home page</a>
```

Comment:
The link for 'Home page' currently uses a placeholder URL ('#'). Consider updating this to point to the actual home page URL.